src/main/java/gov/cms/ab2d/contracts/controller/ContractController.java

```diff
@@ -0,0 +1,42 @@
+package gov.cms.ab2d.contracts.controller;
+
+import gov.cms.ab2d.contracts.model.Contract;
+import gov.cms.ab2d.contracts.service.ContractService;
+import java.util.List;
+import java.util.Optional;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@AllArgsConstructor
+public class ContractController {
+    private final ContractService contractService;
+
+    @GetMapping("/contract/all")
+    public List<Contract> getAllAttestedContracts() {
+        return contractService.getAllContracts();
+    }
+
+    @PostMapping("/contract")
+    public void updateContract(@RequestBody Contract contract) {
+        contractService.updateContract(contract);
+    }
+
+    @GetMapping("/contract")
+    public Contract getContractByValue(@RequestParam("contractId") Optional<Long> contractId, @RequestParam("contractNumber") Optional<String> contractNumber) {
+        if (contractId.isPresent()) {
```

Comment:
This is super picky so feel free to ignore. The logic isn't straightforward to me. I would suggest this:
```
if(id && number)  
      throw exception
      
if(id)
     return ....
else if(number)
     return .....
     
throw exception   // no need for else  
```

Reply:
Ah github didn't auto update so I didn't see @sharshar-sb's comment

Reply:
Fixed issues